examples/README.md

```diff
@@ -5,3 +5,19 @@ The Zarf examples demonstrate different ways to utilize Zarf in your environment
 To test create a virtual area to test all examples, you can run `make all` or `make vm-init` if you've already run the examples before. Run `make vm-destroy` to clean up.
 
 > **NOTE**: Examples are not meant for production use, they exist to demonstrate how to use Zarf in various ways.  Also, examples utilize software pulled from multiple sources and _some_ of them require authenticated access. Check the examples themselves for the specific accounts / logins required.
```

Comment:
Add language about examples being unsupported - and that using them requires knowledge of Kubernetes  and lots of configuration (or something like that)